### DISCUSSION TITLE
Approvals process in Yetiforce CRM??

### DISCUSSION THREAD

**Discussion by cabbinvi** [YetiForceCompany/YetiForceCRM#15597] [How to]
I could not find the seetings of approvals same like Vtiger.  How to configure it? 

![image](https://user-images.githubusercontent.com/96813711/147636552-8d91fbb4-413c-460a-8c8f-fc41d2e74131.png)

**Comment by mariuszkrzaczkowski** [comment]
This mechanism is not available in YetiForce, how does it work?

**Reply by cabbinvi** [reply]
I confused, what is the function of module "Approvals" and "Approvals register"?? are they functioning for asking approvals??
